src/rpcmining.cpp

```diff
@@ -415,8 +416,14 @@ Value getblocktemplate(const Array& params, bool fHelp)
 
     LOCK(cs_main);
 
+    // Wallet is required because we support coinbasetxn
+    if (pwalletMain == NULL) {
+        throw JSONRPCError(RPC_METHOD_NOT_FOUND, "Method not found (disabled)");
+    }
```

Comment:
Should this fall back to not using coinbasetxn if the wallet is disabled? Or maybe it shouldn't even be compiled in that case, like the other wallet-dependent RPCs?